R/ridge_lambda.R

```diff
@@ -0,0 +1,29 @@
+get_lambda_errors <- function(lambdas, X_c, X_t, synth_data, trt, holdout_length=1, scm=T) {
```

Comment:
Can you add a doc string? Check out the other functions for examples of the formatting. After that you can add `devtools::document(<PATH>)` to rebuild the documentation

Reply:
Added.